iib/workers/tasks/build.py

```diff
@@ -724,6 +724,8 @@ def handle_add_request(
     # Check if Gating passes for all the bundles
     if greenwave_config:
         gate_bundles(resolved_bundles, greenwave_config)
+    else:
+        log.warning('Greenwave checks are not active and will be disabled in next release.')
```

Comment:
Should we reword this to something like `Greenwave checks are disabled. Bundles will not be gated`